### PR TITLE
corrected Makefile.am to install headers

### PR DESCRIPTION
**Pull request by mdorier**
@carns That's 2 lines changed but can you still double-check that it fixes https://github.com/mochi-hpc/mochi-margo/issues/179?

### CODE REVIEW

**Comment by roblatham00** [comment]
this is a good start but `make distcheck` is still finding a lot of missing headers.  I'm working on fixing that now.

File: Makefile.am [hunk 1]
```diff
@@ -21,7 +21,9 @@ include_HEADERS = \
  include/margo-logging.h \
  include/margo-diag.h \
  include/margo-bulk-pool.h \
- include/margo-util.h
+ include/margo-util.h \
+ include/margo-bulk-util.h \
+ include/margo-timer.h
```
Comment (roblatham00) [inline Makefile.am:26]: that header lives in `src/margo-timer.h`  maybe it needs to move to include?

**Review by roblatham00 — DISMISSED** [review]

File: Makefile.am (hunk 1 above)
Comment (mdorier) [reply Makefile.am:26]: It does not, there is a margo-timer-private.h in src, margo-timer.h is in include...

Comment (roblatham00) [reply Makefile.am:26]: withdrawn! i forgot to rebase my 'main' onto latest. thanks.

**Comment by roblatham00** [comment]
I may have been a bit too general in which headers are `noinst_HEADERS` but I updated @mdorier 's patch to also get margo passing `make distcheck`

**Comment by carns** [comment]
Thanks @roblatham00 and @mdorier !  This seems right now.

I always forget about `make distcheck`; I should use that more often.

**Pull request merged**